supysonic/schema/migration/mysql/20200620.sql

```diff
@@ -0,0 +1,30 @@
+ALTER TABLE user ADD podcast BOOLEAN DEFAULT false NOT NULL;
+
+CREATE TABLE IF NOT EXISTS podcast_channel (
+    id BINARY(16) PRIMARY KEY,
+    url VARCHAR(256) UNIQUE NOT NULL,
+    title VARCHAR(256),
+    description VARCHAR(256),
+    cover_art VARCHAR(256),
+    original_image_url VARCHAR(256),
+    status TINYINT NOT NULL,
+    error_message VARCHAR(256),
+    created TIMESTAMP NOT NULL,
+    last_fetched TIMESTAMP
+) DEFAULT CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci;
+
+CREATE TABLE IF NOT EXISTS podcast_episode (
+    id BINARY(16) PRIMARY KEY,
+    stream_url VARCHAR(256) NOT NULL,
+    file_path VARCHAR(256),
+    channel_id CHAR(36) NOT NULL,
+    title VARCHAR(256),
+    description VARCHAR(256),
+    duration VARCHAR(8),
+    status TINYINT NOT NULL,
+    publish_date DATETIME,
+    created DATETIME NOT NULL,
+    FOREIGN KEY(channel_id) REFERENCES podcast_channel(id),
```

Comment:
Missed this the first time, there's a type mismatch here: `podcast_episode.channel_id` is `CHAR(36)` while `podcast_channel.id` is `BINARY(16)`.

Reply:
Nice catch! Adding those foreign keys helped highlight the other misses.  😛